From c8417a2b0b66c26eaa0569c19422f87598e29629 Mon Sep 17 00:00:00 2001
From: Mikko Levonmaa <mikko.levonmaa@palm.com>
Date: Tue, 19 Feb 2013 11:16:05 -0800
Subject: [PATCH 1/4] xcbcommon should not be a mandatory dependency

Not all platforms that use qtwayland depend on xcbcommon to be present
for them to be functional

Change-Id: I899d9314bdf772a9d5b8b7c143636f9f12c8ad1d

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 qtwayland.pro | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/qtwayland.pro b/qtwayland.pro
index ee0e5ac..f1a6b6f 100644
--- a/qtwayland.pro
+++ b/qtwayland.pro
@@ -11,15 +11,15 @@ qtCompileTest(xcomposite)
 load(qt_parts)
 
 !config_wayland {
-    error(QtWayland requires Wayland 1.0.0 or higher)
+    error("QtWayland requires Wayland 1.0.0 or higher")
 }
 
 !config_xkbcommon {
-    error(QtWayland requires xkbcommon 0.2.0 or higher)
+    warning("No xkbcommon 0.2.0 or higher found, disabling support for it")
 }
 
 !config_wayland_scanner {
-    error(QtWayland requires wayland-scanner)
+    error("QtWayland requires wayland-scanner")
 }
 
 !config_wayland_egl {
-- 
1.8.2.1

